Dockerfile

```
@@ -1,4 +1,4 @@
-FROM registry.cloudogu.com/official/base:3.18.3-1
+FROM registry.cloudogu.com/official/base:3.20.1-1
```

```suggestion
FROM registry.cloudogu.com/official/base:3.20.1-2
```

Update to use new doguctl v0.12.2